m2cgen/exporters.py

```diff
@@ -158,17 +184,23 @@ def export_to_visual_basic(model, module_name="Model", indent=4):
         The name of the generated module.
     indent : int, optional
         The size of indents in the generated code.
+    function_name : string, optional
+        Name of the function in the generated code (defaults to 'score')
 
     Returns
     -------
     code : string
     """
-    interpreter = interpreters.VisualBasicInterpreter(module_name=module_name,
-                                                      indent=indent)
+    interpreter = interpreters.VisualBasicInterpreter(
+        module_name=module_name,
+        indent=indent,
+        function_name=function_name
+    )
     return _export(model, interpreter)
 
 
-def export_to_c_sharp(model, namespace="ML", class_name="Model", indent=4):
+def export_to_c_sharp(model, namespace="ML", class_name="Model", indent=4,
+                      function_name="Score"):
```

Comment:
< Here and in all other places >
I believe it can be `score` for the consistency. Refer to https://github.com/BayesWitnesses/m2cgen/pull/166#issuecomment-586548983
> I'm not sure that it is worthy of further consideration. As we are still in alpha phase, we are free to change a lot without any deprecation warnings and worrying about the backward compatibility. Especially for just a naming conventions.

Reply:
Hey @StrikerRUS shouldn't we at least comply with basic code style principles of a target language? Like in this case method names are capitalized in C#. I believe this basic logic is worth keeping.

Reply:
I really do not have a strong opinion here, but since C# seems to be using [PascalCasing by definition](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions), I would vote for being consistent with that.

Reply:
To be honest, I don't have a strong opinion on keeping code style for default values. I just proposed the way which is the easiest to maintain 🙂 .
And then we'll need to apply conditional default values for the CLI part: https://github.com/BayesWitnesses/m2cgen/pull/166#issuecomment-586503683.

Reply:
@StrikerRUS All right, I went ahead and implemented the default values for the CLI part such that they directly use the default value of the `function_name` keyword argument.

It relies on inspecting the `export` function (the [inspect module](https://docs.python.org/3.5/library/inspect.html#inspect.signature) introduced in Python 3.5 so that should not be an issue) and might therefore feel a bit hacky, but it allows us to specify the default only once (namely in the keyword argument) and be done with it -- it should automatically down-propagate to the CLI options as well. 

I also added a test which should give us a greater confidence that this works.

If you do not like this approach, I am happy to revert that particular commit and just leave it as it was.

Thanks again!

Reply:
@mrshu Thanks a lot for implementing conditional default values! I like your approach very much! Just please add a comment about that default value is conditional and will be set later based on the `export` function signature.
```
parser.add_argument(
    "--function_name", "-fn", dest="function_name", type=str,
    default=None,  # some comment here 
    help="Name of the function in the generated code.")
```

Reply:
Thanks @StrikerRUS , I just did that.

Please feel free to point out a simpler version of the comment -- I may have overcomplicated it quite a bit.